functions/gsheets.py

```diff
@@ -221,20 +229,16 @@ def lower_case_first_line(iter):
                     f"{row['course code']}: \tbad day value \"{day}\""
                 )
                 continue
-            schedule_data[zoom_series_id]["Days"].add(day)
 
-        schedule_data[zoom_series_id].setdefault("Time", set())
-        time_object = datetime.strptime(row["start"], "%H:%M")
-        schedule_data[zoom_series_id]["Time"].update([
-            datetime.strftime(time_object, "%H:%M")
-        ])
+            schedule_data[zoom_series_id]["events"].append(
+                {"day": day, "time": time, "title": title}
+            )
 
     for item in schedule_data.values():
         # sort days for easier testing
-        item["Days"] = sorted(
-            list(item["Days"]), key=lambda x: valid_days.index(x[0])
+        item["events"] = sorted(
+            list(item["events"]), key=lambda x: valid_days.index(x["day"])
```

Comment:
trying to understand what the `list()` is doing here

Reply:
Was leftover from the previous parsing code. I think it's no longer necessary.